PocketBar/PocketBar/Config.cs

```diff
@@ -6,5 +6,6 @@ namespace PocketBar
 {
 	public static class Config
 	{
+		public const string ApiURL = "https://www.thecocktaildb.com/api/json/v1/1";
```

Comment:
- please put api version on separate variable and use string interpolation to construct apiURL
- rename ApiURL to CocktailAPIURL (we could be using more APIs in the future)